packaging/hudi-cli-bundle/hudi-cli-with-bundle.sh

```diff
@@ -38,9 +38,15 @@ HUDI_CONF_DIR="${DIR}"/conf
 HUDI_AUX_LIB="${DIR}"/auxlib
 
 if [ ! -d $HUDI_AUX_LIB ]; then
-  echo "Downloading necessary auxiliary jars for Hudi CLI"
-  wget https://repo1.maven.org/maven2/org/glassfish/jakarta.el/$JAKARTA_EL_VERSION/jakarta.el-$JAKARTA_EL_VERSION.jar -P auxlib
-  wget https://repo1.maven.org/maven2/jakarta/el/jakarta.el-api/$JAKARTA_EL_VERSION/jakarta.el-api-$JAKARTA_EL_VERSION.jar -P auxlib
+  echo "Downloading necessary auxiliary jars for Hudi CLI to $HUDI_AUX_LIB"
+  wget https://repo1.maven.org/maven2/org/glassfish/jakarta.el/$JAKARTA_EL_VERSION/jakarta.el-$JAKARTA_EL_VERSION.jar -P $HUDI_AUX_LIB
+  wget https://repo1.maven.org/maven2/jakarta/el/jakarta.el-api/$JAKARTA_EL_VERSION/jakarta.el-api-$JAKARTA_EL_VERSION.jar -P $HUDI_AUX_LIB
+
+  if [ "$IS_S3_ENABLED" = "true" ]; then
+    echo "Adding hadoop-aws jars to auxlib"
+    wget https://repo1.maven.org/maven2/com/amazonaws/aws-java-sdk-bundle/1.12.262/aws-java-sdk-bundle-1.12.262.jar -P $HUDI_AUX_LIB
+    wget https://repo1.maven.org/maven2/org/apache/hadoop/hadoop-aws/3.3.4/hadoop-aws-3.3.4.jar -P $HUDI_AUX_LIB
+  fi
```

Comment:
I'm not sure if this makes sense as Hudi already has `hudi-aws` bundle that has those dependencies

Reply:
Agreed @CTTY . I tried with the `hudi-aws` bundle before creating the above fix and that actually didn't work. `s3a` paths required `org.apache.hadoop.fs.s3a.*` package which is defined in the `hadoop-aws` module. `hudi-aws` modules only defines dependency on `hadoop-common`.
Hence, an explicit dependency is required in the `java` classpath.

Lemme know if I should explore this in some other manner.

Reply:
@prabodh1194 : hadoop-aws version needs to align withe HADOOP version installed on the the site where the cli is run. Also, aws-java-sdk-bundle's version needs to align with hadoop-aws version. So, we cannot hard-code the package version here. I would suggest to use an ENV variable HADOOP_VERSION and then instead of using wget, use mvn dependency:get of the specific hadoop-aws version which would also transitively download aws-java-sdk-bundle and get the jar file paths to use.

Reply:
@bvaradar  ACK.

Reply:
@prabodh1194 could you make the change according to @bvaradar 's suggestion?  Also give an example in the script on how the ENV variables can be set and update the website https://hudi.apache.org/docs/cli.